docs/reference/buildx_history_inspect_attachment.md

```diff
@@ -15,3 +15,24 @@ Inspect a build attachment
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+Inspect a specific attachment from a build record, such as a provenance file or
+SBOM. Attachments are optional artifacts stored with the build and may be
+platform-specific.
+
+## Examples
+
+### Inspect a provenance attachment from a build
+
+```console
+docker buildx history inspect attachment mybuild --type https://slsa.dev/provenance/v0.2
+```
+
+### Inspect a SBOM for linux/amd64
+
+```console
+docker buildx history inspect attachment mybuild \
+  --type application/vnd.cyclonedx+json \
+  --platform linux/amd64
+```
```

Comment:
There is also a variant for opening attachment by digest (visible in inpect).

`docker buildx history inspect attachment mybuild sha256:abcdef...`